robot/impl/robot_reconfigure_test.go

```diff
@@ -206,22 +284,132 @@ func TestRobotReconfigure(t *testing.T) {
 		testReconfiguringMismatch = true
 		// processing modify will fail
 		logger := logging.NewTestLogger(t)
-		conf1 := ConfigFromFile(t, "data/diff_config_1.json")
-		conf3 := ConfigFromFile(t, "data/diff_config_4_bad.json")
+		// conf1 := ConfigFromFile(t, "data/diff_config_1.json")
```

Comment:
should these be removed?

Reply:
ah yeah, i just had that as a reference for myself as I was migrating. will remove